### PR TITLE
Added the missing @RobotKeywordOverload. This annotation is needed to…

### PR DESCRIPTION
**Pull request by ractoc**
Added the required annoation to enable the overloading. The only downside of this is that the parameter for the attribute name needs to have an =None declared or it won't work. It is a mandatory parameter though so...

### CODE REVIEW

File: src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java [hunk 1]
```diff
@@ -444,7 +433,7 @@ public String getElementAttribute(String attributeLocator) {
             "Example: ${id}=    Get Element Attribute   css:h1  id\r\n" +
             "\r\n" +
             "Passing attribute name as part of the locator was removed in SeleniumLibrary 3.2. The explicit attribute argument should be used instead.")
-    @ArgumentNames({ "locator", "attribute" })
+    @ArgumentNames({ "locator", "attribute=None" })
```
Comment (YauheniPo) [inline src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: remove '=None'

Comment (YauheniPo) [reply src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: use robot.... and default value "None"

Comment (YauheniPo) [inline src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: My approach:
@ArgumentNames({ "locator", "attribute=None" })
public String getElementAttribute(String locator, String... attribute) {
String attribute= robot.getParamsValue(attribute, 0, "None");

What do you think?

Comment (ractoc) [reply src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: I tried this but for some reason it wouldn't except both the version with 1 parameter and the version with 2 parameters that way. That would only work with just the String... params parameter.

Comment (ractoc) [reply src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: Ah wait, serves me right, trying to answer stuff like this without access to the code... See the fix below.

File: src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java [hunk 2]
```diff
@@ -444,14 +433,15 @@ public String getElementAttribute(String attributeLocator) {
             "Example: ${id}=    Get Element Attribute   css:h1  id\r\n" +
             "\r\n" +
             "Passing attribute name as part of the locator was removed in SeleniumLibrary 3.2. The explicit attribute argument should be used instead.")
-    @ArgumentNames({ "locator", "attribute" })
-    public String getElementAttribute(String locator, String attribute) {
+    @ArgumentNames({ "locator", "attribute=None" })
+    public String getElementAttribute(String locator, String... attribute) {
+        String attributeName= robot.getParamsValue(attribute, 0, "None");
```
Comment (YauheniPo) [inline src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: String attributeName = -formating

File: src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java (hunk 1 above)
Comment (YauheniPo) [reply src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: Thanks, great

File: src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java (hunk 2 above)
Comment (ractoc) [reply src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: What do you mean exactly?

Comment (YauheniPo) [reply src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: String attributeName = robot.getParamsValue(attribute, 0, "None");
                                 ^

Comment (ractoc) [reply src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java]: ah found it, space between attributeName and = sign. 
And fixed...

**Comment by ractoc** [comment]
Can you give any indication when this will be available through mvncentral?

**Pull request merged**